src/main/resources/api-definition.yaml

```diff
@@ -10,6 +10,13 @@ documentation:
       content: Welcome to the OpenLMIS fulfillment-service documentation.
 
 schemas:
+  - basicOrderDto: !include schemas/basicOrderDto.json
+  - basicOrderDtoPage: !include schemas/orderDtoPage.json
```

Comment:
this is a typo: I create a new PR with fix for that: https://github.com/OpenLMIS/openlmis-fulfillment/pull/9